package.json

```diff
@@ -11,9 +11,9 @@
 		"email": "admin@tri.be"
 	},
 	"browserslist": [
-		"last 2 versions",
-		"not op_mini all",
-		"not dead"
+		"defaults",
```

Comment:
Simpler [set of browsers](https://browsersl.ist/#q=defaults%2C%0Anot+ios+%3C+17.5&region=alt-na) to remember while still supporting the older versions of Safari we need.